sdk/servicebus/azure_messaging_servicebus/src/models/mod.rs

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) Microsoft Corporation. All Rights reserved
+// Licensed under the MIT license.
+
+//! Models and types used throughout the Service Bus client.
+
+use azure_core::fmt::SafeDebug;
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+
+/// Represents the state of a Service Bus entity.
+#[derive(Clone, Copy, SafeDebug, PartialEq, Eq, Serialize, Deserialize)]
+#[non_exhaustive]
+pub enum EntityState {
+    /// The entity is active and can send/receive messages.
+    #[serde(rename = "Active")]
```

Comment:
Nit: all these renames are unnecessary if you just use `#[serde(rename_all = "PascalCase")]` similar to what you did on line 31 below.